input/postgres/statements.go

```diff
@@ -235,13 +236,14 @@ func GetStatements(server *state.Server, logger *util.Logger, db *sql.DB, global
 		if showtext {
 			statementTexts[key] = receivedQuery.String
 		}
-		if queryID.Valid && showtext {
+		if queryID.Valid {
```

Comment:
Is this change related to using the cache below?

It seems that if you kept the `&& showtext` it would work just fine, since we are only using `statementFingerprints` in that situation.

Or was the motivation here to bump up the LastSeen (when not on showtext) to increase the chance we keep the mapping around for when the next full snapshot occurs?

Reply:
It would be good to bump the `LastSeen` during compact snapshots but that's not required for this PR so I'll remove it.